bin/ebuild-helpers/portageq

```diff
@@ -13,7 +13,7 @@ set -f # in case ${PATH} contains any shell glob characters
 
 for path in ${PATH}; do
 	[[ -x ${path}/${scriptname} ]] || continue
-	[[ ${path} == ${PORTAGE_OVERRIDE_EPREFIX}/usr/lib*/portage/* ]] && continue
+	[[ ${path} == */portage/bin/ebuild-helpers* ]] && continue
```

Comment:
Generally the path looks like `/usr/lib/portage/python3.7/ebuild-helpers`, so maybe use `*/portage/*/ebuild-helpers/*`.